### ISSUE TITLE
Statistic Item: Gold background hides focus indicator

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#100]
While not a failure, the gold background on stats hides the gold focus indicator on links. 

Try to change the focus indicator color on this background if possible
